### PR TITLE
Document multiple dependencies in pub-add

### PR DESCRIPTION
**Pull request by sigurdm**
Fix of: https://github.com/dart-lang/pub/issues/3450

### CODE REVIEW

**Comment by sigurdm** [comment]
Yeah - really we should fix it. It feels so stupid to write it out...

Added an explanation.

File: src/tools/pub/cmd/pub-add.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ $ dart pub add --dev test
 For options that apply to all pub commands, see
 [Global options](/tools/pub/cmd#global-options).
 
+The options given will apply to all the packages added in an invocation. For
+example `dart pub add test http --dev` will add both package:test and
+package:http as dev-dependencies.
```
Comment (parlough) [inline src/tools/pub/cmd/pub-add.md:43]: Some minor adjustments to be consistent with the rest of the page and make it a note to better fit with the flow of the page:

```suggestion
{{site.alert.note}}
  Any specified options will apply to all the packages
  included in an invocation of the command.
  For example, `dart pub add test http --dev` 
  will add both the `test` and `http` packages 
  as dev dependencies.
{{site.alert.end}}
```

File: src/tools/pub/cmd/pub-add.md [hunk 2]
```diff
@@ -6,14 +6,14 @@ description: Use dart pub add to add a dependency.
...
-This command adds the specified package to the pubspec as a dependency,
-and then gets the dependency.
-For example, the following command is equivalent to
-editing `pubspec.yaml` to add the `http` package,
-and then calling `dart pub get`:
+This command adds the specified packages to the pubspec as dependencies, and
+then gets the dependencies.
+
+For example, the following command is equivalent to editing `pubspec.yaml` to
+add the `http` package, and then calling `dart pub get`:
```
Comment (parlough) [inline src/tools/pub/cmd/pub-add.md:16]: Adjusting the semantic breaks:

```suggestion
This command adds the specified packages to the pubspec as dependencies, 
and then gets the dependencies.

For example, the following command is equivalent to
editing `pubspec.yaml` to add the `http` package, 
and then calling `dart pub get`:
```

**Review by parlough — APPROVED** [review]
Some minor formatting adjustments, but looks good to me. Thanks!

**Pull request merged**